src/main/java/com/example/artvswar/config/AppConfig.java

```diff
@@ -36,7 +35,7 @@ public void addCorsMappings(CorsRegistry registry) {
     }
 
     @Bean
-    public RestTemplate getRestTemplate() {
-        return new RestTemplateBuilder().build();
+    public WebClient buildWebClient() {
```

Comment:
Oh, I was meaning to use 'RestClient' but if you want we can do it with WebClient.